### DISCUSSION TITLE
挂载百度网盘，文件视频都无法打开

### DISCUSSION THREAD

**Discussion by Acacia-Ma** [AlistGo/alist#6842] [Q&A]
2024/07/22 00:38:34.487278 ERROR RESTY json: cannot unmarshal number into Go struct field DownloadResp.request_id of type string

我使用百度网盘挂载，是svip，使用官方下载接口，打开了web代理，修改了ua为pan.baidu.com之后，文件夹加载什么都正常，就是 文件和视频都打不开，会出现上述报错，请问如何解决？

**Comment by Acacia-Ma** [comment]
> 2024/07/22 00:38:34.487278 ERROR RESTY json: cannot unmarshal number into Go struct field DownloadResp.request_id of type string
> 
> 我使用百度网盘挂载，是svip，使用官方下载接口，打开了web代理，修改了ua为pan.baidu.com之后，文件夹加载什么都正常，就是 文件和视频都打不开，会出现上述报错，请问如何解决？

版本是最新版

**Reply by Threeth0** [reply]
应该是百度网盘账号出现问题了。
我在百度网盘中打开视频在线观看时弹出来需要申诉，并且在电脑端百度网盘下载时需要更改密码，改一下密码重新登陆就好了。
记得更新一下alist百度网盘配置中的刷新令牌。
我的配置改成这样了，不需要改UA也可以看
<img width="980" alt="截屏2024-08-25 20 54 14" src="https://github.com/user-attachments/assets/462a3a9c-608c-4176-8beb-946de35788cf">
<img width="971" alt="截屏2024-08-25 20 55 18" src="https://github.com/user-attachments/assets/56f88792-dd17-4670-b097-397b7e3a507d">
